lib/Ohmbrewer_Heating_Element.cpp

```diff
@@ -50,6 +104,7 @@ Ohmbrewer::HeatingElement::~HeatingElement() {
  */
 int Ohmbrewer::HeatingElement::doWork() {
     // TODO: Implement HeatingElement::doWork
+    //digitalWrite( someWayOfGettingTheSinglePin(), (getState() : HIGH ? LOW) )
```

Comment:
That's kinda my point... It hasn't been merged to master yet. So we're going to merge this PR and it'll get in master. Then you're going to work on the previous PR and it's probably going to say you've got a conflict because this line was changed in both or something (especially if you rebase to get the new stuff from master into that existing PR).